src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -218,6 +223,9 @@ export const generateContent: TranslationFn = content => {
   const displayState = currentStateFn(userCase).at(
     (userCase.state === State.OfflineDocumentReceived ? userCase.previousState : userCase.state) as State
   );
+
+  const isFinalOrderComplete = userCase.state === State.FinalOrderComplete;
```

Comment:
maybe update the name to isFinalOrderCompleteState

Reply:
Renamed constant as suggested